### ISSUE TITLE
Rpi 1

### ISSUE THREAD

**Issue by hexabyte23** [luis-guilherme/rpi-cross-compile-docker#1]
Thanks for this great doc, I am looking for it since a while !

I unfortunately only have RPI 1, is there any tips for it ?

**Comment by luis-guilherme** [comment]
Not really, this is just a fork I did from someone else. But I'll probably look into it too, as I am interested too
